_candidates_en/TaketoIkeda_1.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Taketo Ikeda
+title: "For Expressions Revisited Part2: On My 0wn"
```

Comment:
Note: this title was provided by the OP so I didn't touch it (except adding `:`)